packages/core/lib/check/checker.js

```diff
@@ -121,14 +121,17 @@ class Checker {
       return result;
     }
 
-    const nullish = Checker.checkNullish(reqHeaders);
-    const nullishError = nullish.error;
-    const nullishSimilarity = nullish.similarity;
-    if (nullish) {
-      result.type = "HEADERS";
-      result.error = nullishError;
-      result.similarity = nullishSimilarity;
-      return result;
+    console.log(options.skipCheckHeaderValueNullable)
```

Comment:
thank you !! could you remove this log ?

Reply:
Ooops... Fixed it!